src/code/audio/code_8019AF00.c

```diff
@@ -2071,38 +2071,41 @@ void AudioOcarina_MemoryGameSetNumNotes(u8 minigameRound) {
 
 /**
  * Unused remnant of OoT, Id 14 now represent Goron Lullaby Intro instead of the OoT ocarina memory game
+ *
  */
-#define OCARINA_SONG_MEMORYGAME OCARINA_SONG_GORON_LULLABY_INTRO
+#define OCARINA_SONG_MEMORYGAME 14
 s32 AudioOcarina_MemoryGameGenerateNotes(void) {
-    u32 rndButtonIdx;
-    u8 rndPitch;
+    u32 randButtonIndex;
+    u8 randPitch;
 
-    if (sOcaMemoryGameAppendPos == sOcaMemoryGameEndPos) {
+    if (sOcarinaMemoryGameAppendPos == sOcaMemoryGameEndPos) {
         return true;
     }
 
-    rndButtonIdx = Audio_NextRandom();
-    rndPitch = sButtonToPitchMap[rndButtonIdx % 5];
+    randButtonIndex = Audio_NextRandom();
+    randPitch = sButtonToPitchMap[randButtonIndex % ARRAY_COUNT(sButtonToPitchMap)];
 
-    if (sOcarinaSongNotes[OCARINA_SONG_GORON_LULLABY_INTRO][sOcaMemoryGameAppendPos - 1].pitch == rndPitch) {
-        rndPitch = sButtonToPitchMap[(rndButtonIdx + 1) % 5];
+    if (sOcarinaSongNotes[OCARINA_SONG_MEMORYGAME][sOcarinaMemoryGameAppendPos - 1].pitch == randPitch) {
+        randPitch = sButtonToPitchMap[(randButtonIndex + 1) % ARRAY_COUNT(sButtonToPitchMap)];
     }
 
-    sOcarinaSongNotes[OCARINA_SONG_GORON_LULLABY_INTRO][sOcaMemoryGameAppendPos].pitch = rndPitch;
-    sOcarinaSongNotes[OCARINA_SONG_GORON_LULLABY_INTRO][sOcaMemoryGameAppendPos].length = 45;
-    sOcarinaSongNotes[OCARINA_SONG_GORON_LULLABY_INTRO][sOcaMemoryGameAppendPos].volume = 0x50;
-    sOcarinaSongNotes[OCARINA_SONG_GORON_LULLABY_INTRO][sOcaMemoryGameAppendPos].vibrato = 0;
-    sOcarinaSongNotes[OCARINA_SONG_GORON_LULLABY_INTRO][sOcaMemoryGameAppendPos].bend = 0;
+    //! @bug calling this function in MM will overwrite Goron Lullaby Intro
```

Comment:
If it's a guaranteed consequence of calling the function, I feel like it should be noted more prominently outside it.

Reply:
This is my understanding from reading the code, although I have not verified this in game. I'll test this and get back to this comment after that

Reply:
Yeah, I mean in the sense that you're only going to see this comment if you actually read the function contents. If someone just read the comment above it, they wouldn't know it would cause problems if left unmodified.